pkgs/applications/misc/kratos/default.nix

```diff
@@ -0,0 +1,45 @@
+{ fetchFromGitHub, buildGoModule, lib, stdenv }:
+
+buildGoModule rec {
+  pname = "kratos";
+  version = "v0.6.0-alpha.1.pre.2";
+
+  src = fetchFromGitHub {
+    owner = "ory";
+    repo = "kratos";
+    rev = "fa7fa701051c819ac74c919bc52721c57f662fb8";
+    sha256 = "1jyhb5cs532msqfgnhn6bnnqw6d94g3pjgirklbrysd3z4f32f8g";
+  };
+
+  vendorSha256 = "1n5g6mrr5c8z6y3ncpkap971n6m5qimpfqkq8vjgl8jkih4m282x";
+  buildPhase = ''
+    runHook preBuild
+
+    # Patch shebangs
+    files=(
+       Makefile
+       test/e2e/run.sh
+       script/testenv.sh
+       script/test-envs.sh
+       persistence/sql/migratest/update_fixtures.sh
+    )
+    for i in "''${files[@]}"; do
+      patchShebangs "$i"
+    done
+
+    # patchShebangs doesn't work for this Makefile, do it manually
+    substituteInPlace Makefile --replace '/bin/bash' '${stdenv.shell}'
+
+    GO111MODULE=on
+    make install
+
+    runHook postBuild
+  '';
```

Comment:
Hmm, can you check if adding the following instead of overriding `buildPhase` produces the same result?
```
  subPackages = [ "." ];

  buildFlags = [ "-tags sqlite" ];

  preBuild = ''
    # Patch shebangs
    ...
    # patchShebangs doesn't work for this Makefile, do it manually
    ...
  '';
```

Reply:
It actually does, thank you!